src/main/kotlin/org/rust/cargo/project/module/util/RustCrateUtil.kt

```diff
@@ -76,7 +76,42 @@ private val Module.externCrates: Collection<ExternCrate> get() =
             contentRoot.findFileByRelativePath(crate.path)
 
         vFile?.let { ExternCrate(crate.name, it) }
+    } + sdkCrates
+
+object SdkCrates {
+    const val std: String = "std"
+    const val core: String = "core"
+}
+
+private val Module.sdkCrates: Collection<ExternCrate> get() {
+    val src = locateRustSources() ?: return emptyList()
+
+    return listOf(
+        "std" to "libstd/lib.rs",
+        "core" to "libcore/lib.rs",
+        "collections" to "libcollections/lib.rs"
+    ).mapNotNull {
+        val (crateName, srcLocation) = it
```

Comment:
That's a shame Kotlin is incapable to unbind tuples inside the lambdas. Though the team promised that'd be fixed by 1.1.